### DISCUSSION TITLE
PostCSS / Bootstrap / Parcel Question and best way to compile? *First Time Parcel*

### DISCUSSION THREAD

**Discussion by foxxtay** [parcel-bundler/parcel#6183] [Q&A]
Hey! I am new to Parcel and getting things set up. I'm a little confused by an issue I am seeing with PostCSS and Bootstrap. When I set up my project and saved normal sass being imported into my .js entry point, PostCSS works just as expected. However, when I import Bootstrap to the js or try and import the Bootstrap minified css to my scss file, the entire library is installed along side of it. Is there a reason PostCSS doesn't handle imports like that? Or is there something I am missing that I need to include. My postcss.config.js is very simple, as you can view below:

`module.exports = {
	plugins: [
		require("postcss-uncss")({
			html: [
				'./index.php'
			]
		})
	]
}`

I followed along a very simple tutorial to get that set-up with my files, but wasn't able to find an answer on why imports wouldn't be compiling as well.

**Comment by DeMoorJasper** [comment]
Not entirely sure what you mean, but if I understand correctly you'd like us to create common chunks of css node_modules, that's currently not implemented as far as I know. Also sass is pretty much a black box when it comes to controlling how it works, so not even sure if it's possible with sass.
